backends/conrod_wgpu/examples/all_winit_wgpu.rs

```diff
@@ -306,9 +302,9 @@ fn create_logo_texture(
     let cmd_encoder_desc = wgpu::CommandEncoderDescriptor {
         label: Some("conrod_upload_image_command_encoder"),
     };
-    let mut encoder = device.create_command_encoder(&cmd_encoder_desc);
-    encoder.copy_buffer_to_texture(buffer_copy_view, texture_copy_view, extent);
-    queue.submit(&[encoder.finish()]);
+    let encoder = device.create_command_encoder(&cmd_encoder_desc);
+    queue.write_texture(texture_copy_view, data, data_layout, extent);
```

Comment:
This change is needed because wgpu now is a bit stricter about the data alignment so we'll see this error otherwise:

```
thread 'main' panicked at 'bytes per row does not respect `COPY_BYTES_PER_ROW_ALIGNMENT`', /home/yutani/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.6.0/src/backend/direct.rs:1323:35
```